test/WebJobs.Script.Tests/EndToEndTestsBase.cs

```diff
@@ -44,5 +48,87 @@ public EndToEndTestsBase(TTestFixture fixture)
             Assert.True(trace.Contains(TestHelpers.RemoveByteOrderMarkAndWhitespace("script processed queue message")));
             Assert.True(trace.Contains(TestHelpers.RemoveByteOrderMarkAndWhitespace(messageContent)));
         }
+
+        protected async Task EasyTablesTest(bool writeToQueue = true)
```

Comment:
I explicitly didn't want them running on the other platforms until I had a chance to test them -- that's why only node/.NET right now. Everything else was lower pri.

I don't *think* there's anything preventing this from working in the other scripting languages. Maybe I'll try them today and if they just work, then I'll leave it in.

---
In reply to: [56192241](https://github.com/Azure/azure-webjobs-sdk-script/pull/98#discussion_r56192241) [](ancestors = 56192241)